src/aliceVision/camera/Equidistant.cpp

```diff
@@ -270,21 +264,28 @@ Eigen::Matrix<double, 3, 2> Equidistant::getDerivativetoUnitSphereWrtScale(const
 
 Eigen::Matrix<double, 3, Eigen::Dynamic> Equidistant::getDerivativeBackProjectUnitWrtParams(const Vec2& pt2D) const 
 {
-    size_t disto_size = getDistortionParamsSize();
-
     const Vec2 ptMeters = ima2cam(pt2D);
     const Vec2 ptUndist = removeDistortion(ptMeters);
     const Vec3 ptSphere = toUnitSphere(ptUndist);
 
-    Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic> J(3, getParams().size());
+    Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic> J(3, getParameters().size());
 
     J.block<3, 2>(0, 0) = getDerivativetoUnitSphereWrtScale(ptUndist);
     J.block<3, 2>(0, 2) = getDerivativetoUnitSphereWrtPoint(ptUndist) * getDerivativeRemoveDistoWrtPt(ptMeters) * getDerivativeIma2CamWrtPrincipalPoint();
 
-    if (disto_size > 0)
-    {
-        J.block(0, 4, 3, disto_size) = getDerivativetoUnitSphereWrtPoint(ptUndist) * getDerivativeRemoveDistoWrtDisto(ptMeters);
-    }
+    return J;
+}
+
+Eigen::Matrix<double, 3, Eigen::Dynamic> Equidistant::getDerivativeBackProjectUnitWrtDistortion(const Vec2& pt2D) const
+{
+    size_t disto_size = getDistortionParamsSize();
```

Comment:
```suggestion
    size_t distoSize = getDistortionParamsSize();
```

Also, out of scope, but since "getParameters()" has been renamed, it might be worth it to rename "getDistortionParamsSize()" to "getDistortionParametersSize()" in the future as well.